src/classes/CON_ContactMerge_TDTM.cls

```diff
@@ -105,93 +112,253 @@ public class CON_ContactMerge_TDTM extends TDTM_Runnable {
-        	if (con.Account.npe01__SYSTEM_AccountType__c == CAO_Constants.HH_ACCOUNT_TYPE)
-        	   listAccIdHH.add(con.AccountId);
-            if (con.Account.npe01__SYSTEM_AccountType__c == CAO_Constants.HH_ACCOUNT_TYPE ||
-                con.Account.npe01__SYSTEM_AccountType__c == CAO_Constants.ONE_TO_ONE_ORGANIZATION_TYPE ||
-                con.Account.npe01__SYSTEM_AccountType__c == CAO_Constants.BUCKET_ORGANIZATION_TYPE) {
-                mapContactIdAccountIdWinner.put(con.Id, con.AccountId);        
+
+        /*******************************************************************************************************
+        * @description Returns the account Id of accounts with household account types
+        ********************************************************************************************************/
+        @TestVisible
+        private List<Id> getHouseholdAcctIds(List<Account> accounts) {
+            List<Id> accountIds = new List<Id>();
+
+            for (Account record : accounts) {
+                if (isHousehold(record.npe01__SYSTEM_AccountType__c)) {
+                    accountIds.add(record.Id);
+                }
+            }
+
+            return accountIds;
+        }
+
+        /*******************************************************************************************************
+        * @description Returns the account Id of the winning contacts accounts with household account types
+        ********************************************************************************************************/
+        @TestVisible
+        private List<Id> getHouseholdAcctIds(List<Contact> contacts) {
+            List<Id> accountIds = new List<Id>();
+
+            for (Contact record: contacts) {
+                if (isHousehold(record.Account.npe01__SYSTEM_AccountType__c)) {
+                    accountIds.add(record.AccountId);
+                }
             }
+
+            return accountIds;
         }
-        
-        // we only want to move opps from non-org accounts, so we need to query for their type.
-        map<Id, Account> mapAccountIdAccountLoser = new map<Id, Account>([select Id, npe01__SYSTEM_AccountType__c 
-            from Account where Id in :listAccountIdLoser]);
-        for (integer i = 0; i < listContactIdLoser.size(); i++) {
-        	if (listAccountIdLoser[i] != null) {
-                Account accLoser = mapAccountIdAccountLoser.get(listAccountIdLoser[i]);
-                if (accLoser == null)
+
+        /*******************************************************************************************************
+        * @description Adds the account Id of the losing contacts account when it
+        * is any type of individual account
+        ********************************************************************************************************/
+        @TestVisible
+        private Map<Id, Id> getOldAcctIdForOppsByWinningContactId(List<Account> losersAccounts) {
```

Comment:
Not sure how to change the name so it is shorter but still descriptive. These suggestions are too long: `extractOldIndividualAccountIdsByMasterContactId()`, or `mapOldIndividualAccountIdsToMasterContactIds()`.  Or can it be just `extractOldIndividualAccountIds()` while the result containing `Map` var can be more descriptive?